### DISCUSSION TITLE
Permision and credit needed?

### DISCUSSION THREAD

**Discussion by Newtown97** [Scythe-Anticheat/Scythe-Anticheat#96] [General]
Hello there! I would like to use Scythe Anti-Cheat for my soon to be public server. Do I need permision to use this on my server or credit? Thank you for your time!

**Comment by MrDiamond64** [comment]
Anyone can use this on their realm or server. You don't need to give credit either.

**Comment by ExtraBytez** [comment]
Is it bc it has a credit function built in?

**Comment by ExtraBytez** [comment]
Answer there is a 
```
/function credit
```
Built in
